.gitignore

```
@@ -35,15 +35,14 @@ yarn-error.log*
 *.tsbuildinfo
 next-env.d.ts
 
-### yarn ###
-# used Zero-Install
-.yarn/*
-.yarn/cache/*
-.yarn/releases/*
-!.yarn/patches/*
-!.yarn/plugins/*
-!.yarn/sdks/*
-!.yarn/versions/*
+# yarn with no zero-install
+# https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
 .pnp.*
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
+!.yarn/sdks
+!.yarn/versions
```

```suggestion
.yarn/*
!.yarn/cache
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```
생각해보니 의존성도 git을 이용하여 버전 관리를 한다면 Zero-Install 의 장점을 가져갈 수 있었는데 추가하는건 어떻게 생각하시나요?

---

오 그럼 zero-install로 변경해둘게요~